src/analysis/complex/basic.lean

```diff
@@ -258,6 +258,26 @@ lemma isometry_of_real : isometry (coe : ℝ → ℂ) := of_real_li.isometry
 /-- Continuous linear map version of the canonical embedding of `ℝ` in `ℂ`. -/
 def of_real_clm : ℝ →L[ℝ] ℂ := of_real_li.to_continuous_linear_map
 
+/-- The only continuous ring homomorphisms from `ℂ` to `ℂ` are the identity and the complex
+conjugation, -/
+lemma ring_hom_continuous (f : ℂ →+* ℂ) (hf : continuous f) :
```

Comment:
The name should be something like `ring_hom_eq_or_eq_of_continuous` or `ring_hom_eq_id_or_conj_of_continuous`.